### PR TITLE
Restructure code

### PR DESCRIPTION
**Pull request by ytsek**

### CODE REVIEW

File: Gigamon_ThreatINSIGHT_module_type.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        {
+            "id": "observe",
+            "description": "Query Gigamon ThreatINSIGHT for IP, Domain, MD5, SHA1, and SHA256 return Sighting and Indicator."
+        },
+        {
+            "id": "refer",
+            "description": "Pivot on IP, Domain, MD5, SHA1, and SHA256 to the Gigamon ThreatINSIGHT UI."
+        }
+    ],
+    "properties": {
```
Comment (AnastasiiaRozlyvan) [inline Gigamon_ThreatINSIGHT_module_type.json:65]: Please, add "url": "https://ciscohosted.url" to `properties` as it is done in other integrations, for example [here ](https://github.com/CiscoSecurity/tr-05-serverless-pulsedive/blob/2c65c04786951c3e4aaa419c4a2486e3844885fb/module_type.json.sample#L48).
And  it is better to rename this file into `module_type.json.sample`.

File: .travis.yml [hunk 2]
```diff
@@ -1,23 +1,11 @@
...
-- coverage run --source api/ -m pytest --verbose tests/unit/ && coverage report
-- pytest --verbose tests/functional/
+services:
+- docker
+jobs:
+  include:
+  - stage: build&test
+    script:
+    - docker build -t tr-05-serverless-gigamon-threatinsight .
+    - docker run -d -p 9090:9090 --name tr-05-serverless-gigamon-threatinsight tr-05-serverless-gigamon-threatinsight
```
Comment (AnastasiiaRozlyvan) [inline .travis.yml:8]: Let's rename `tr-05-serverless-gigamon-threatinsight` into `tr-05-gigamon-threatinsight` everywhere in this PR as this integration won't be serverless anymore.

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env sh
+echo " .:|:.:|:. "
+echo " C I S C O "
+echo "  SecureX "
+echo
+echo " Development Dockerfile build script."
+echo
+
+module_name="Gigamon ThreatINSIGHT Relay"
```
Comment (AnastasiiaRozlyvan) [inline build.sh:9]: Please, rename `Gigamon ThreatINSIGHT Relay` -> `Gigamon ThreatINSIGHT`

File: Gigamon_ThreatINSIGHT_module_type.json (hunk 1 above)
Comment (ytsek) [reply Gigamon_ThreatINSIGHT_module_type.json:65]: URL added, like in sample.

File: build.sh (hunk 3 above)
Comment (ytsek) [reply build.sh]: Relay mentions removed.

File: .travis.yml (hunk 2 above)
Comment (ytsek) [reply .travis.yml]: All the mentions of `tr-05-serverless-gigamon-threatinsight` renamed into `tr-05-gigamon-threatinsight`.

File: code/README.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [![Gitter Chat](https://img.shields.io/badge/gitter-join%20chat-brightgreen.svg)](https://gitter.im/CiscoSecurity/Threat-Response "Gitter Chat")
 [![Travis CI Build Status](https://travis-ci.com/CiscoSecurity/tr-05-serverless-gigamon-threatinsight.svg?branch=develop)](https://travis-ci.com/CiscoSecurity/tr-05-serverless-gigamon-threatinsight)
```
Comment (ytsek) [inline code/README.md:2]: @AnastasiiaRozlyvan What about this line?

Comment (AnastasiiaRozlyvan) [reply code/README.md:2]: This line is ok.  It's link to our travis-ci build

File: README.md [hunk 5]
```diff
@@ -1,132 +1,61 @@
...
-To `deploy` your application to AWS as a Lambda function for the first time,
-run the following command:
+## Testing (Optional)
+If you want to test the application you will require Docker and several dependencies from the [requirements.txt](code/requirements.txt) file:
 ```
-zappa deploy dev
+pip install --upgrade --requirement code/requirements.txt
 ```
+You can perform two kinds of testing:
+- Run static code analysis checking for any semantic discrepancies and [PEP 8](https://www.python.org/dev/peps/pep-0008/) compliance:
```
Comment (AnastasiiaRozlyvan) [inline README.md:24]: Oh, I see some Object Replacement Characters in this README. Can we fix this?
Screenshot 2021-03-16 at 09 02 26

Comment (ytsek) [reply README.md:24]: I have already removed Object Replacement Characters you mentioned.

**Review by AnastasiiaRozlyvan — APPROVED** [review]

**Review by AlinaBlahun — COMMENTED** [review]
Is watchdog endpoint covered with tests?

File: Gigamon_ThreatINSIGHT_module_type.json [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
...he token; the token is not retrievable once the dialog is closed.\n4. Deploy [Gigamon ThreatINSIGHT Serverless Relay](https://github.com/CiscoSecurity/tr-05-gigamon-threatinsight#gigamon-threatinsight-relay-api)\n5. Complete the **Add New Gigamon ThreatINSIGHT Module** form:\n    - **Module Name** - Leave the default name or enter a name that is meaningful to you.\n    - **URL** - The URL of the Gigamon ThreatINSIGHT Serverless Relay.\n    - **API KEY** - Enter the Gigamon ThreatINSIGHT API Key.\n    - **CTR ENTITIES LIMIT** - Enter the limit that restricts the maximum number of CTIM entities of each type returned in a single response per each requested observable. Must be a positive integer. Defaults to 100 (if unset or incorrect). Has the upper bound of 1000 to avoid getting overwhelmed with too much data, so any greater values are still acceptable but also limited at the same time.\n    - **GTI_ALLOW_TEST_ACCOUNTS** - Allows fake data from the test accounts (Demo and Training) to be returned along with real data (if enabled). Must be 1 - if allow or 0 - if not. Defaults to 0 (if unset or incorrect).\n6. Click **Save** to complete the Gigamon ThreatINSIGHT module configuration.",
```
[the remark this answers is not in the document]
Comment (ytsek) [reply Gigamon_ThreatINSIGHT_module_type.json]: Already removed.

[the remark this answers is not in the document]
Comment (AnastasiiaRozlyvan) [reply Gigamon_ThreatINSIGHT_module_type.json]: @ytsek sorry and step 4 in tips is deprecated `4. Deploy [Gigamon ThreatINSIGHT Serverless Relay](https://github.com/CiscoSecurity/tr-05-gigamon-threatinsight#gigamon-threatinsight-relay-api)`

[the remark this answers is not in the document]
Comment (ytsek) [reply Gigamon_ThreatINSIGHT_module_type.json]: @AnastasiiaRozlyvan step 4 in tips already removed

**Comment by ytsek** [comment]
> Is watchdog endpoint covered with tests?

@AlinaBlahun Already covered

**Review by TarasMaliuchenko — APPROVED** [review]

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**